ui/desktop/app/routes/scopes/scope/projects/sessions/index.js

```diff
@@ -136,27 +136,21 @@ export default class ScopesScopeProjectsSessionsIndexRoute extends Route {
     const chunkedTargetIds = chunk(uniqueTargetIds, 50);
 
     const associatedTargetsPromises = chunkedTargetIds.map((targetIds) =>
-      this.store.query(
-        'target',
-        {
-          scope_id: orgScope.id,
-          recursive: true,
-          force_refresh: true,
-          query: {
-            filters: {
-              id: {
-                logicalOperator: 'or',
-                values: targetIds.map((targetId) => ({
-                  equals: targetId,
-                })),
-              },
+      this.store.query('target', {
+        scope_id: orgScope.id,
+        recursive: true,
+        force_refresh: true,
+        query: {
+          filters: {
+            id: {
+              logicalOperator: 'or',
+              values: targetIds.map((targetId) => ({
+                equals: targetId,
+              })),
             },
           },
         },
-        {
-          pushToStore: false,
```

Comment:
I made the decision to just push all the targets from this request to the store. It _should_ be fine since it's limited to the user's sessions so there shouldn't be that many.

We could alternatively just not push to the store for this request still, and then make a separate request to indexed db for the current page's sessions and just push those targets to the store but I'm assuming for the majority of cases that's just an extra request.

Reply:
Yeah I think this is a safe solution. Likely a single user isn't going to have hundreds of thousands of sessions that still exist so the performance shouldn't be hurt.